### ISSUE TITLE
ComboBox validation error template

### ISSUE THREAD

**Issue by zsonglorman** [Kinnara/ModernWpf#168]
Hello,

I would like my combo box to have a red border and a tooltip upon validation error. I'm using the following style:

```xml
<Style TargetType="{x:Type ComboBox}" BasedOn="{StaticResource DefaultComboBoxStyle}">
   <Style.Triggers>
      <Trigger Property="Validation.HasError" Value="True">
         <Setter Property="ToolTip" Value="{Binding RelativeSource={RelativeSource Self}, Path=(Validation.Errors)[0].ErrorContent}" />
         <Setter Property="Validation.ErrorTemplate">
            <Setter.Value>
               <ControlTemplate>
                  <DockPanel>
                     <Border BorderBrush="Red" BorderThickness="2">
                        <AdornedElementPlaceholder />
                     </Border>
                  </DockPanel>
               </ControlTemplate>
            </Setter.Value>
         </Setter>
      </Trigger>
   </Style.Triggers>
</Style>
```

It's strange, because the tooltip is working as expected, but I cannot make it work to show the red border around the combo box. If I use a similar style for textboxes, it works correctly. Also, if I remove the _BasedOn_ part in the style above, the red border appears, but then the combo box has a very basic outlook.

What am I doing wrong? Thank you in advance.

**Comment by Kinnara** [comment]
Thanks for reporting. Looks like there is a bug in DefaultComboBoxStyle that causes the ErrorTemplate to only show up if the ComboBox is editable (`IsEditable="True"`). I'll try to fix it in the near future. In the meantime, a workaround is to add `<Setter Property="Validation.ValidationAdornerSiteFor" Value="{Binding RelativeSource={RelativeSource Self}}" />` to the style.

**Comment by Kinnara** [comment]
Fixed by 2e25969.
